transaction.go

```diff
@@ -100,9 +101,13 @@ type payloadCanonicalForm struct {
 	Authorizers               [][]byte
 }
 
+type transactionSignatureCommonForm interface {
+	dummy()
```

Comment:
Use a descriptive name that is only implemented by intended types, e.g.

```suggestion
	isTransactionSignatureCommonForm()
```